quarkus/README.md

```diff
@@ -0,0 +1,60 @@
+# Agent Check: Quarkus
+
+## Overview
+
+This check monitors [Quarkus][1] through the Datadog Agent. 
+
+Include a high level overview of what this integration does:
```

Comment:
I know you plan to update the README later -- I still don't recommend leaving placeholders in the copy, since it's too easy to accidentally publish them. Most of this README looks fine, I just recommend deleting lines 7-10.

Reply:
👌 I understand! Will do.